cmd/jaeger/features/main.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2023 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
```

Comment:
The requirement is to make it a sub command of jaeger binary, not a standalone binary.